fixtures/uitests/tests/ui/non_hashable_record_key.stderr

```diff
@@ -1,60 +1,14 @@
-error[E0277]: the trait bound `f32: std::cmp::Eq` is not satisfied
- --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
-  |
-  | uniffi::deps::static_assertions::assert_impl_all!(f32: ::std::cmp::Eq, ::std::hash::Hash); // record<f32, u64>
-  |                                                   ^^^ the trait `std::cmp::Eq` is not implemented for `f32`
-  |
-  = help: the following other types implement trait `std::cmp::Eq`:
-            i128
-            i16
-            i32
-            i64
-            i8
-            isize
-            u128
-            u16
-          and $N others
-note: required by a bound in `assert_impl_all`
- --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
-  |
-  | uniffi::deps::static_assertions::assert_impl_all!(f32: ::std::cmp::Eq, ::std::hash::Hash); // record<f32, u64>
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `assert_impl_all`
-  = note: this error originates in the macro `uniffi::deps::static_assertions::assert_impl_all` (in Nightly builds, run with -Z macro-backtrace for more info)
-
-error[E0277]: the trait bound `f32: Hash` is not satisfied
- --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
-  |
-  | uniffi::deps::static_assertions::assert_impl_all!(f32: ::std::cmp::Eq, ::std::hash::Hash); // record<f32, u64>
-  |                                                   ^^^ the trait `Hash` is not implemented for `f32`
-  |
-  = help: the following other types implement trait `Hash`:
-            i128
-            i16
-            i32
-            i64
-            i8
-            isize
-            u128
-            u16
-          and $N others
-note: required by a bound in `assert_impl_all`
- --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
-  |
-  | uniffi::deps::static_assertions::assert_impl_all!(f32: ::std::cmp::Eq, ::std::hash::Hash); // record<f32, u64>
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `assert_impl_all`
-  = note: this error originates in the macro `uniffi::deps::static_assertions::assert_impl_all` (in Nightly builds, run with -Z macro-backtrace for more info)
-
-error[E0425]: cannot find function `get_dict` in this scope
- --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
-  |
-  |     <std::collections::HashMap<f32, u64> as uniffi::FfiConverter<crate::UniFfiTag>>::lower(r#get_dict())
-  |                                                                                            ^^^^^^^^^^ not found in this scope
-
 error[E0277]: the trait bound `f32: Hash` is not satisfied
  --> $OUT_DIR[uniffi_uitests]/records.uniffi.rs
   |
-  |     <std::collections::HashMap<f32, u64> as uniffi::FfiConverter<crate::UniFfiTag>>::lower(r#get_dict())
-  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Hash` is not implemented for `f32`
+  | / pub extern "C" fn r#uitests_82f4_get_dict(
```

Comment:
isn't this hash here and below going to be a problem?

Reply:
I could be missing it, but I don't see why.  The error message seems okay to me, what else do we need to watch out for?

Reply:
I assumed it meant that these `.stderr` files would now require changing each release?

Reply:
Ahhh, I see your point now and that would be very annoying.  Maybe it's not going to be an issue, since I need to change the checksum code anyways.  Now that the proc-macros don't know the metadata, they can't include the checksum in the function name.  Instead, they'll have to export that value as a separate symbol that the bindings code checks.  I think the askama templates should do the same, which hopefully will fix this.